src/service/standaloneServer.ts

```diff
@@ -34,7 +34,7 @@ export async function startStandaloneServer(
   const logger = options.logger.child({ service: 'cortex-backend' });
 
   const config = await loadBackendConfig({ logger, argv: process.argv });
-  const discoveryApi = SingleHostDiscovery.fromConfig(config);
+  const discovery = HostDiscovery.fromConfig(config);
```

Comment:
`SingleHostDiscovery` is deprecated so I went ahead and updated to use `HostDiscovery`